docs/src/main/paradox/intellij-idea.md

```diff
@@ -0,0 +1,26 @@
+IntelliJ IDEA
+-------------
+
+[IntelliJ](https://www.jetbrains.com/idea/) from JetBrains is one of the leading IDEs in the Java/Scala community, and it has excellent support for Akka HTTP. This section will guide you through setting up, testing and running the sample project.
+
+## Setting up the project
+
+Open IntelliJ and select File -> Open... and point to the directory where you have installed the sample project. There should be a pop-up like this:
+
+![Open Project](images/idea-open-project.png)
+
+Fill out the settings according to the above and press `OK` to import the project. If IntelliJ will warn about missing Scala SDK, it is only to follow the instructions to add support.
+
+## Inspecting the code
+
+If we open up the file `src/main/scala/com/lightbend/akka/http/sample/QuickstartServer.scala` we can see a lot of lines beginning with //# .... These lines are used as directives for this documentation. To get rid of these lines from the source code we can utilize the awesome Find/Replace functionality in IntelliJ. Select Edit -> Find -> Replace in Path.... Check the Regex box and add the following regex [//#].* and click on Replace in Find Window.... Select to replace all occurrences and voila the lines are gone!
```

Comment:
Do we really need this? Seems like a distraction...

Wrap [//#].* in backticks.